File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int i,j,n;
    while(1){
        n = get_int("Height: ");
        if(n>=1 && n<=8){
            break;
        }
    }
    for(i=1;i<=n;i++){
        for(j=1;j<=n-i;j++){
            printf(" ");
        }
        for(j=1;j<=i;j++){
            printf("#");
        }
        printf("\n");
    }
}

File: sentimental-mario-less/mario.py
from cs50 import get_int

# Prompt the user for a positive integer between 1 and 8 (inclusive)
while True:
    n = get_int("Height: ")
    if n >= 1 and n <= 8:
        break

# Print the pyramid
for i in range(n):
    # Print spaces
    for j in range(n - i - 1):
        print(" ", end="")

    # Print left hashes
    for j in range(i + 1):
        print("#", end="")
        
    print()
